src/dotnet-bootstrapper/dotnet-bootstrapper.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>dotnet-bootstrapper</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <SelfContained>true</SelfContained>
+    <SignAssembly>true</SignAssembly>
+    <TargetFramework>net8.0</TargetFramework>
+    <RunAOTCompilation>true</RunAOTCompilation>
+    <PublishTrimmed>true</PublishTrimmed>
+    <IsPackable>false</IsPackable>
+    <PublishSingleFile>true</PublishSingleFile>
+    <PublishAot>true</PublishAot>
+    <WasmStripILAfterAOT>true</WasmStripILAfterAOT>
+    <RollForward>LatestMajor</RollForward>
```

Comment:
We should review which of these properties should be set.  I don't think `RollForward` applies to a self-contained app, and I don't think `WasmStripILAfterAOT` applies to non-WASM projects.  Some of the other AOT / publish properties may be redundant, I think the main one you need to specify is `PublishAOT`.